internal/manager/featuregates/feature_gates.go

```diff
@@ -33,6 +33,10 @@ const (
 	// See: https://docs.konghq.com/gateway/latest/key-concepts/routes/expressions/ about expression based routes in Kong 3.0+.
 	ExpressionRoutesFeature = "ExpressionRoutes"
 
+	// CombinedServicesFeature is the name of the feature-gate that makes KIC create a single Kong Service when
+	// a Kubernetes Service is referenced by multiple netv1.Ingresses. It's effective only when CombinedRoutesFeature is enabled.
+	CombinedServicesFeature = "CombinedServices"
```

Comment:
Seems that the PR did not add the code to apply the feature gate. Are you going to leave this for another PR?

Reply:
That's a fair point, thanks for catching this 😅 a6f9a53